packages/voice/src/models/v1/dice-request/dice-request.ts

```diff
@@ -9,22 +9,23 @@ import { CallHeader } from '../call-header';
 import { DiceRequestDebit } from '../dice-request-debit';
 import { DiceRequestUserRate } from '../dice-request-user-rate';
 import { IceRequestTo } from '../ice-request-to';
+import { GetCallResponseObjReasonEnum, GetCallResponseObjResultEnum } from '../enums';
 
 /**
  * The request body of a Disconnected Call Event.
  */
 export interface DiceRequest {
 
   /** Must have the value `dice`. */
-  event?: string;
+  event?: 'dice';
   /** The unique ID assigned to this call. */
   callId?: string;
   /** The timestamp in UTC format. */
   timestamp?: Date;
   /** The reason the call was disconnected. */
-  reason?: string;
+  reason?: GetCallResponseObjReasonEnum;
```

Comment:
What about more generalized name for enum? e.g.
```suggestion
  reason?: CallResultReasonEnum;
```